dev/src/db_utils.py

```diff
@@ -1,83 +1,38 @@
-from typing import Any, cast
+from typing import Any
 
-from psycopg import Cursor, sql
+from src.python_sql import Column, Table
```

Comment:
Better would be `from .python_sql import ...`

Reply:
done